plugins/codeamp/release_handler.go

```
@@ -66,9 +66,11 @@ func (x *CodeAmp) ReleaseEventHandler(e transistor.Event) error {
 						releaseExtension.SecretsSignature, releaseExtension.FeatureHash,
 						[]string{"complete"}).Order("created_at desc").First(&lastReleaseExtension).Error
 					if err != nil {
+						log.Error(err.Error())
```

You should return the error as well.

---

not here because we still want to continue execution if a release extension isn't found